helix-core/src/test/java/org/apache/helix/monitoring/mbeans/TestDropResourceMetricsReset.java

```diff
@@ -191,6 +191,10 @@ public void onMBeanRegistered(MBeanServerConnection server,
     @Override
     public void onMBeanUnRegistered(MBeanServerConnection server,
         MBeanServerNotification mbsNotification) {
+      System.out.println("onUnRegistered: " + _objectName +
```

Comment:
Can we use log instead of system print out?

Reply:
in test, we don't have logger. I saw other examples in the test code as well.

Reply:
We do have logs for test. Those system print out are the ancient legacy code. We did not refactor it.

Better we add the loggers there.

Reply:
used logger.